katello/katello-debug.sh

```diff
@@ -33,6 +33,9 @@ add_files /etc/foreman-installer/*
 add_cmd "find /root/ssl-build -ls | sort -k 11" "katello_ssl_build_dir"
 add_cmd "find /etc/pki -ls | sort -k 11" "katello_pki_dir"
 
+# Installation and upgrade times
+add_cmd "egrep '(katello|satellite)' /var/log/yum.log" "yum_install_upgrade_log"
```

Comment:
Should `satellite` be a downstream only change?

Reply:
Do we have this for Foreman packages? and plugins?

Reply:
I am fine removing "satellite" here. @ehelms not sure I get what you mean tho?